src/main.cpp

```diff
@@ -95,6 +99,9 @@ int main(int argc, char *argv[])
     }
 
     std::vector<unsigned char> bytecode;
+    // TODO - Allow reading debug code from different source for in the event we are not
```

Comment:
Can you make this comment shorter? I prefer to keep TODO comments to one line, so that a grep shows the whole message. Suggest `// TODO: Read debug info from file if needed.`.